javascript/javascript1/week2/class07Students.js

```diff
@@ -0,0 +1,52 @@
+// copy your homework solution here to make the webpage work!
+
+const class07Students = [];
+let count = 0;
+function addStudentToClass(studentName) {
+   if (studentName.toLowerCase() == 'queen') {
+class07Students.push(studentName);
+  }
+  else if ((class07Students.indexOf(studentName) !== -1) || (studentName == '' ) ){
+    return 'Cannot add same students to class 07';
+ }
```

Comment:
There's an array function called `.includes` that helps with this.
```javascript
if(class07Students.includes(studentName)) {
  console.log(`Student ${studentName} is already in the class`);
```

Note: the console.log needs to say the student's name